python/ray/experimental/serve/global_state.py

```diff
@@ -100,27 +101,50 @@ def init_or_get_http_server(self,
         if "http_server" not in self.actor_handle_cache:
             [handle] = ray.get(
                 self.actor_nursery_handle.start_actor.remote(
-                    HTTPActor, init_args=(), tag="http_server"))
+                    HTTPActor, init_args=(), init_kwargs={},
```

Comment:
Let's can add default for init_args and init_kwargs in start_actor definition so we don't need to write boilerplate here. 
(but this requires either tag moved before init_args or default tag to None and check its value, preferring the first one)